Tests/SonarScanner.MSBuild.Shim.Test/SonarScannerWrapperTests.cs

```diff
@@ -245,28 +256,42 @@ public void SonarScanner_UserSpecifiedEnvVars_OnlySONARSCANNEROPTSIsPassed()
         }
 
         [TestMethod]
-        public void WrapperError_Success_NoStdErr()
-        {
+        public void WrapperError_Success_NoStdErr() =>
             TestWrapperErrorHandling(executeResult: true, addMessageToStdErr: false, expectedOutcome: true);
-        }
 
         [TestMethod]
-        [WorkItem(202)] //SONARMSBRU-202
-        public void WrapperError_Success_StdErr()
-        {
+        [WorkItem(202)] // SONARMSBRU-202
+        public void WrapperError_Success_StdErr() =>
             TestWrapperErrorHandling(executeResult: true, addMessageToStdErr: true, expectedOutcome: true);
-        }
 
         [TestMethod]
-        public void WrapperError_Fail_NoStdErr()
-        {
+        public void WrapperError_Fail_NoStdErr() =>
             TestWrapperErrorHandling(executeResult: false, addMessageToStdErr: false, expectedOutcome: false);
-        }
 
         [TestMethod]
-        public void WrapperError_Fail_StdErr()
-        {
+        public void WrapperError_Fail_StdErr() =>
             TestWrapperErrorHandling(executeResult: false, addMessageToStdErr: true, expectedOutcome: false);
+
+        [TestMethod]
+        public void FindScannerExe_ReturnsScannerCliBat()
+        {
+            // Act
+            var scannerCliScriptPath = SonarScannerWrapper.FindScannerExe();
+
+            // Assert
+            scannerCliScriptPath.Should().EndWithEquivalentOf(@"sonar-scanner-4.8.0.2856\bin\sonar-scanner.bat");
```

Comment:
We should not hardcode the version in the string, as it will fail once we update the sonar-scanner-cli.